tests/core/v5_1/test_dispatcher_session_management.py

```diff
@@ -1,29 +1,60 @@
+import secrets
+
 from eth_enr import OldSequenceNumber
 import pytest
 import trio
 
+from ddht.base_message import AnyOutboundMessage
 from ddht.v5_1.constants import SESSION_IDLE_TIMEOUT
+from ddht.v5_1.messages import PingMessage
 
 
 @pytest.mark.trio
-async def test_dispatcher_intiator_and_recipient_detects_handshake_timeout(
-    driver, autojump_clock
+async def test_dispatcher_detects_handshake_timeout_as_initiator(
+    alice, bob, alice_client, bob_client, autojump_clock
 ):
-    await driver.initiator.send_ping()
+    await bob_client.manager.stop()
```

Comment:
The *correct* way to do this externally on a service is `bob_client.get_manager().stop()` but this is almost exclusively only for type checking reasons.

Otherwise, maybe instead of stopping bob, you could just not include the `bob_client` fixture in this test which I think would have the same effect.

Reply:
For some reason, the test becomes a little flaky when leaving out `bob_client` and explicitly stopping `bob_client` solves the flake

Reply:
In that case, you should instead create a new `enr` record and simply try to connect to it (since it won't have a running service listening on whatever port it generates).